### PR TITLE
Add interoperability test suite to CI

### PR DESCRIPTION
**Pull request by marinthiercelin**

### CODE REVIEW

File: .github/workflows/sop-test-suite.yml
Comment (larabr) [inline .github/workflows/sop-test-suite.yml:84]: Idk if it's temporary that it's pointing to the main branch of ProtonMail/openpgp-interop-test-analyzer, but I think it'd be best to create and point to tags (e.g. v1) instead, so that if we need to change something on the main branch, we don't have to worry about it being usable immediately by the workflows

File: .github/workflows/sop-test-suite.yml
Comment (marinthiercelin) [reply .github/workflows/sop-test-suite.yml]: Good point, I removed the push trigger

File: .github/workflows/sop-test-suite.yml
Comment (marinthiercelin) [reply .github/workflows/sop-test-suite.yml]: Right, I added a v1 tag

**Review by larabr — APPROVED** [review]

**Comment by marinthiercelin** [comment]
I don't have write access to the repo, so I can't merge it.

**Pull request merged**
